.github/workflows/ci_pipeline_bazel.yml

```diff
@@ -0,0 +1,35 @@
+name: CI pipeline (Bazel)
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build-test:
+    runs-on: ${{ matrix.image }}
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - ubuntu-22.04
+          - ubuntu-22.04-arm
```

Comment:
Just a post question: Why 22? Why not 24? How do we decide what we support for bazel? We run the colcon CI on 22, but because we build on docker images of other versions as well, so that choice can be more or less arbitrary, but here we run "bare-metal".

Reply:
I think this will do: https://github.com/Ekumen-OS/beluga/pull/521/commits/8a52ba2aae20ca67010d950ebffa37b1bf6f2987